### ISSUE TITLE
(Self-Voicing) Reconsider delivery of object and context responses now that we have utterance queue 

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#249]
From our sprint meeting on John Travoltage on Friday, October 30th and @emily-phet's follow-up comments in this doc https://phetsims.slack.com/files/U6DER46F3/F01DZ4WP22Y/jt-and-gflb-notes.docx

We need to reconsider the delivery of object and context responses now that we have finer control with the utterance queue, and we are using the Self-Voicing feature with non-visual users.

There are a couple of things to consider here as we start to broaden the potential audience who would be using this feature.

1. How much do we want to increase the amount of output being delivered in the current design for which we already had a few interviews which indicates that the design was working nicely for people who had trouble reading the screen or seeing some details on the screen. We carefully considered the inputs when deciding where and what is delivered.
    - alt input & gesture input - information delivered on focus
    - mouse & touch -  information delivered on intentional interaction (click & touch)

2. We have to keep in mind we already have layers of content that we can turn off and on:
    - Object changes and on-screen text
    - Context changes
    - Helpful hints

3. With 1 & 2 in mind, what kinds of options would we want to create that would enhance or change the current design to specifically improve access for non-visual learners or a screen-less experience?

**Comment by jessegreenberg** [comment]
We addressed these considerations in changes made during #251.
